lib/midi-smtp-server/tls-transport.rb

```diff
@@ -43,7 +43,15 @@ def initialize(cert_path, key_path, ciphers, methods, cert_cn, cert_san, logger)
         raise "File \”#{@cert_path}\" does not exist or is not a regular file. Could not load certificate." unless File.file?(@cert_path.to_s)
         raise "File \”#{@key_path}\" does not exist or is not a regular file. Could not load private key." unless File.file?(@key_path.to_s)
         # try to load certificate and key
-        @ssl_context.cert = OpenSSL::X509::Certificate.new(File.open(@cert_path.to_s))
+        cert_lines = File.read(@cert_path.to_s).lines
+        cert_indexes = cert_lines.each_with_index.map { |line, index| index if line.downcase.include?('begin cert') }.compact
+        certs = []
+        cert_indexes.each_with_index do |cert_index, current_index|
+          end_index = (cert_indexes[current_index + 1] || 0) - 1
```

Comment:
please change into:

end_index = ( current_index + 1 < cert_indexes.length ) ? cert_indexes[current_index + 1] : -1

Reply:
changed and fixed with rubocop 👍